core/src/main/resources/org/apache/spark/log4j2-defaults.properties

```diff
@@ -0,0 +1,55 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# Set everything to be logged to the console
+rootLogger.level = warn
+rootLogger.appenderRef.stdout.ref = STDOUT
+
+appender.console.type = Console
+appender.console.name = STDOUT
+appender.console.target = SYSTEM_OUT
```

Comment:
Random thing I just started noticing, the default logging now goes to stdout instead of stderr, was that intentional? The `log4j2.properties.template` uses `stderr` too.

Reply:
https://github.com/apache/spark/blob/78a5825fe266c0884d2dd18cbca9625fa258d7f7/core/src/main/resources/org/apache/spark/log4j-defaults.properties#L18-L23

https://github.com/apache/spark/blob/78d492c1b153240dddc636ec6002e7bfc6b94b3b/core/src/main/resources/org/apache/spark/log4j2-defaults.properties#L19-L26

look different, WDYT? @viirya

Reply:
Might be when I copied across properties file and missed to change.

Reply:
roughly scanned, should be the only this place changes from stderr to stdout, https://github.com/apache/spark/pull/37854 try fix this